packages/python-pulpcore/python-pulpcore.spec

```diff
@@ -1,15 +1,12 @@
-%{!?_root_bindir:%global _root_bindir %{_bindir}}
-%{!?_root_libexecdir:%global _root_libexecdir %{_libexecdir}}
 %global __python3 /usr/bin/python3.12
 %global python3_pkgversion 3.12
 
 # Created by pyp2rpm-3.3.3
 %global pypi_name pulpcore
-%global wrappers gunicorn pulpcore-worker pulp-content pulpcore-manager
```

Comment:
- `gunicorn` unused since https://github.com/theforeman/puppet-pulpcore/commit/7e12c0f4c5e7552d1aa7350c960b5c46d0130bbf
- `pulpcore-worker` unused since https://github.com/theforeman/puppet-pulpcore/pull/383
- I can't find any reference that we ever used the `pulp-content` (we used `gunicorn` directly until the switch to 3.39) or `pulpcore-manager` (https://github.com/theforeman/puppet-pulpcore/blob/master/manifests/admin.pp never had `libexec` in it's `$PATH`) wrappers